README.md

```diff
@@ -133,6 +138,15 @@ $tag = preg_replace('/attrauthcomanage:/','yourthememodule:', $this->data['e'],
 replacing ```yourthememodule``` with the name of your theme module. 
 Also you must copy the ```attrauthcomanage.definition.json``` file under `yourthememodule/dictionaries` and then change the error messages in order to override the defaults.
 
+### State Attributes
+The filter adds the following attributes to the state information that SimpleSAMLphp stores in association with the request:
+```php
+$state['rciamAttributes'] => array(
+    'registryUserId' => // User's COPerson ID 
+    'aup' => array( ...
+)
```

Comment:
this is incomplete. can you add the $coPersonId (int) and the actual aup model or a reference to models/AUP.md?